docs/proposals/tool-to-backfill-tsdb-blocks.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Tools to backfill TSDB blocks"
```

Comment:
This doesn't sound a good title for a design doc... LOL

Reply:
```suggestion
title: "Backfill TSDB blocks"
```